hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleSignTest.java

```diff
@@ -581,9 +581,10 @@ final Stream<DynamicTest> retestsActivationOnSignWithEmptySigMap() {
                 getAccountBalance(SENDER).hasTinyBars(664L));
     }
 
-    @LeakyHapiTest(overrides = {"ledger.schedule.txExpiryTimeSecs"})
+    @LeakyHapiTest(overrides = {"ledger.schedule.txExpiryTimeSecs", "scheduling.longTermEnabled"})
     final Stream<DynamicTest> signFailsDueToDeletedExpiration() {
         return hapiTest(
+                overriding("scheduling.longTermEnabled", "false"),
```

Comment:
Why is this required?

Reply:
This config is only used when the longTerm is disabled:
```
        final long maxLifetime = schedulingConfig.longTermEnabled()
                ? schedulingConfig.maxExpirationFutureSeconds()
                : ledgerConfig.scheduleTxExpiryTimeSecs();
```
So this test only works when we have this config set to **false**.

Reply:
The test's name suggests that it checks if signing fails once a scheduled transaction expires. I would expect the same behavior once long-term scheduled transactions are enabled. Would the test pass if we also set `schedulingConfig.maxExpirationFutureSeconds` to 1?

Reply:
Good point! I found that we have a couple of places that we are using directly `ledgerConfig.scheduleTxExpiryTimeSecs` instead of checking if `longTermEnabled`. These should be fixed. But I think there is something more there 🤔 We will investigate

Reply:
We have a tests in the new suites that covers if a ScheduleSignTransaction fails after a schedule is expired. The new tests don't override the config but use the new property expiringAt to set a given expiration time. I think its fine to keep this test as it is and leave it to test only the behavior when the flag is off.